api-samples/push/README.md

```diff
@@ -0,0 +1,23 @@
+# self.registration.pushManager.subscribe() userVisibleOnly
+
+This sample demonstrates using the [Push API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) in chrome with `self.registration.pushManager.subscribe()` and specifically how to use `userVisibleOnly` to silence required notifications when receiving a push message in a service worker based extension.
+
+## Overview
+
+By calling a method in the sample and using an external push server website we can simulate an extension receiving a push message where it is required to emit a notification and where it can bypass that requirement (`userVisibleOnly = false`).
+
+## Running this extension
+
+1. Clone this repository.
+2. Go to [web-push-codelab.glitch.me](web-push-codelab.glitch.me) and copy the “Public Key” to the `applicationServerPublicKey` variable in background.js.
+3. Load this directory in Chrome as an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
```

Comment:
Could we add a note that on macOS (and maybe other OS') you need to enable notifications for Google Chrome in system preferences? If you don't do that you never see a notification.